gateway/ln-gateway/src/db.rs

```diff
@@ -285,6 +307,8 @@ pub struct FederationConfig {
     pub timelock_delta: u64,
     #[serde(with = "serde_routing_fees")]
     pub fees: RoutingFees,
+    #[serde(with = "serde_routing_fees")]
+    pub transaction_fees: RoutingFees,
```

Comment:
Remove the RoutingFee struct here, consistently used PaymentFee everywhere instead. Ideally remove RoutingFee completely.

Reply:
I removed as much I reasonably could. There's two reasons completely removing it is difficult:

1) Its currently in DB structs necessary for db migrations
2) It is used in the `LightningGateway` struct that is also persisted on the server side, and I don't think doing a guardian db migration is worth it.

Right now `PaymentFee` is used everywhere except in some db migrations and the LNv1 code. So once LNv1 is removed it will be gone.